Team417_PowerPlay/src/main/java/org/firstinspires/ftc/team417_PowerPlay/InterleagueAutoLeft.java

```diff
@@ -0,0 +1,69 @@
+package org.firstinspires.ftc.team417_PowerPlay;
+
+import com.acmerobotics.roadrunner.geometry.Pose2d;
+import com.acmerobotics.roadrunner.trajectory.Trajectory;
+import com.qualcomm.robotcore.eventloop.opmode.Autonomous;
+
+@Autonomous(name="Interleague Left")
+public class InterleagueAutoLeft extends BaseAutonomous {
+    @Override
+    public void runOpMode() throws InterruptedException {
+        initializeAuto();
+
+        // declare trajectories here
+        Pose2d startPose = new Pose2d(0, 0, Math.toRadians(0));
+
+        Trajectory traject1 = drive.trajectoryBuilder(startPose, false)
```

Comment:
I wonder if you could give this trajectories more informative variable names rather than just numbering them.

Reply:
Makes sense!